src/main/io/ledstrip.h

```diff
@@ -17,6 +17,9 @@
 
 #pragma once
 
+#include <string.h>
+#include "common/color.h"
```

Comment:
Don't need to include `common/color.h` - instead forward declare `struct hsvColor_s` and use that.